Formula/scio.rb

```diff
@@ -0,0 +1,26 @@
+class Scio< Formula
+  desc "Scala API for Google Cloud Dataflow and Apache Beam"
+  homepage "https://github.com/spotify/scio/"
+  url "https://github.com/spotify/scio/releases/download/v0.2.0/scio-repl-0.2.0.jar"
+  sha256 "220e03dfd866d8475fc7943f60376ea4fe0862c3520674e46ef8262a09d1756c"
+
+  bottle :unneeded
+
+  def install
+    libexec.install "scio-repl-#{version}.jar"
+    (bin+"scio-repl").write <<-EOS.undent
+      #!/bin/bash
+      java_opts=()
+      scio_opts=()
+      for o in "$@"; do
+          if [[ "$o" =~ ^-D.* ]]; then
+              java_opts+=("$o")
+          else
+              scio_opts+=("$o")
+          fi
+      done
+
+      exec java ${java_opts[*]} -jar #{libexec}/"scio-repl-#{version}.jar" ${scio_opts[*]}
+    EOS
+  end
+end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!